features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/TARGET_NUCLEO_F439ZI/mbedtls_device.h

```diff
@@ -21,6 +21,7 @@
 #define MBEDTLS_DEVICE_H
 
 #define MBEDTLS_AES_ALT
+#define MBEDTLS_SHA1_ALT
 
-
+#define MBEDTLS_SHA1_C
```

Comment:
This is an application specific definition, and should be defined by the end user. Please remove it